PowerShell/README.md

```diff
@@ -0,0 +1,52 @@
+# BitTitan PowerShell
+This folder contains a set of tools, building blocks and full scenario scripts tailored to work with the BitTitan PowerShell SDK. 
+
+## Quick Startup
+1. Install the [BitTitan PowerShell SDK](https://www.bittitan.com/downloads/bittitanpowershellsetup.msi).
```

Comment:
In the future, we plan to make SDK available via gallery, correct? I imagine installation can be moved to a separate section with couple of options: install via MSI, install via gallery.

Reply:
Cool, then what I'll do is create an Installation section and change this step to say something like "Install the BitTitan PowerShell SDK. The installation section details the steps to install the SDK.".